### DISCUSSION TITLE
COMP Tips使用规范

### DISCUSSION THREAD

**Discussion by WPCJATH** [ZHANG-CAIQI/PolyU_COMP#54] [COMP Tips]
## 使用规范
1. 请不要使用辱骂、误导、反动性质的词汇或言论
2. 请不要上载课程课件、试卷、作业等涉及版权的内容
3. 允许向自己的私人公众号引流，但请不要是商业广告性质的公众号
4. 可以参照[示例](https://github.com/ZHANG-CAIQI/PolyU_COMP/discussions/51)
5. 如有其他疑问或建议，可以WeChat联系WPCJATH、pppxav，或email联系mdwxz@icloud.com、xavier.pan@connect.polyu.hk。谢谢配合！

**Comment by hanshanglin** [comment]
> 2\. 设计

typo

**Reply by WPCJATH** [reply]
thanks
